README.md

```diff
@@ -1,25 +1,32 @@
 # FIWARE IoT Agent Framework
 
 [![License badge](https://img.shields.io/badge/license-AGPL-blue.svg)](https://opensource.org/licenses/AGPL-3.0)
-[![Documentation badge](https://readthedocs.org/projects/fiware-iotagent-node-lib/badge/?version=latest)](http://fiware-iotagent-ul.readthedocs.org/en/latest/?badge=latest)
+[![Documentation badge](https://readthedocs.org/projects/iotagent-node-lib/badge/?version=latest)](http://iotagent-node-lib.readthedocs.org/en/latest/?badge=latest)
 [![Support badge]( https://img.shields.io/badge/support-sof-yellowgreen.svg)](http://stackoverflow.com/questions/tagged/fiware)
 
 ## Index
 
 * [Overview](#overview)
-* [Build & Install](#buildinstall)
-* [API Overview](#apioverview)
-  * [About API](#aboutapi)
-  * [Device Provisioning API](#provisioningapi)
-  * [Configuration API](#configurationapi)
-* [Advanced Topics](#advancedtopics)
-  * [Secured access to the Context Broker](#securedaccess)
-  * [Data mapping plugins](#datamapping)
-  * [Old IoTAgent data migration](#datamigration)
-* [Testing](#librarytesting)
-* [Development Documentation](#development)
-
-## <a name="overview"/> Overview
+  + [Description](#description)
+  + [Device to NGSI Mapping](#device-to-ngsi-mapping)
+  + [Features](#features)
+  + [The ´TimeInstant´ element](#the-%C2%B4timeinstant%C2%B4-element)
```

Comment:
Broken link. It seems that

```
#the-timeinstant-element
```

solves it.

In addition, maybe `´TimeInstant´` should be changed to 

```
`TimeInstant`
```

I guess that the original idea was to use "code style" for that, but probably the author use the wrong symbol :)

Reply:
Fixed in 5e4e34e